internal/db/schema/migrations/oss/postgres/29/04_kms_keys.up.sql

```diff
@@ -0,0 +1,213 @@
+begin;
+
+-- make the required schema changes to adopt
+-- github.com/hashicorp/go-kms-wrapping/extras/kms/v2 
+-- this migration is from:
+-- https://github.com/hashicorp/go-kms-wrapping/blob/main/extras/kms/migrations/postgres/04_keys.up.sql 
+
+-- we can use our existing kms_root_key table.
+
+-- we need to add an add additional constraint to the kms_root_key_version
+-- table: 
+-- https://github.com/hashicorp/go-kms-wrapping/blob/c06f9db9380f2c26cad07cf51f7a324d25dd55ba/extras/kms/migrations/postgres/04_keys.up.sql#L29 
+alter table kms_root_key_version
+  add constraint not_empty_key
+  check (
+      length(key) > 0
+    );
+
+
+create table kms_data_key (
+  private_id kms_private_id primary key,
+  root_key_id kms_private_id not null
+    references kms_root_key(private_id)
```

Comment:
Can you name this foreign key constraint with a name that follows our standard foreign key naming convention? Should be `kms_root_key_fkey` for this one.

Reply:
In my opinion this should use the same foreign key name as the migration in [kms](https://github.com/hashicorp/go-kms-wrapping/blob/main/extras/kms/migrations/postgres/04_keys.up.sql) to make it easier to compare. 

This could be something we try to address in the library, if we want to follow the same naming conventions there.

Reply:
done!